### DISCUSSION TITLE
Can I write a plugin in private repo?

### DISCUSSION THREAD

**Discussion by xzycn** [coredns/coredns#5546] [Q&A]
Hi, it's my first time to write plugin for coredns, I haven't see the way to write a plugin. I thought it just like a chrome extension.

Anyway, I found the way write external plugin(out of tree),but we should write it's url of public git repo, like **log:github.com/coredns/coredns/plugin/log** in plugin.cfg.
How about a private repo?

**Comment by chrisohaver** [comment]
Yes it can be in a private repo. Syntax is the same.
